+mexopencv/make.m

```diff
@@ -436,3 +434,16 @@ function check_path_mexopencv(opts)
         end
     end
 end
+
+%%
+%% Return: true if the environment is Octave.
+%%
+function retval = isOctave
+    persistent cacheval;  % speeds up repeated calls
+
+    if isempty (cacheval)
+        cacheval = (exist ("OCTAVE_VERSION", "builtin") > 0);
```

Comment:
Just noticed that and changed. Thanks.

2015-06-12 11:42 GMT-03:00 Amro <notifications@github.com>:

> In +mexopencv/make.m
> <https://github.com/kyamagu/mexopencv/pull/165#discussion_r32321528>:
>
> > @@ -436,3 +434,16 @@ function check_path_mexopencv(opts)
> >          end
> >      end
> >  end
> > +
> > +%%
> > +%% Return: true if the environment is Octave.
> > +%%
> > +function retval = isOctave
> > +    persistent cacheval;  % speeds up repeated calls
> > +
> > +    if isempty (cacheval)
> > +        cacheval = (exist ("OCTAVE_VERSION", "builtin") > 0);
>
> you cant have double-quoted strings in MATLAB (only Octave), you gotta
> make them single-quotes...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kyamagu/mexopencv/pull/165/files#r32321528>.
>